### PR TITLE
Feature/#15 create overview page

### PR DESCRIPTION
**Pull request by LisaPMunich**

### CODE REVIEW

File: apps/client/src/app/app.module.ts [hunk 1]
```diff
@@ -25,7 +30,9 @@ import { CommonModule } from '@angular/common';
         HttpClientModule,
         CommonModule,
     ],
-    providers: [],
+    providers: [
+        { provide: LOCALE_ID, useValue: 'de' }
```
Comment (Macavity) [inline apps/client/src/app/app.module.ts:34]: There is an environment variable you can use from the file `.env`: "APP_LOCALE"

File: apps/client/src/app/pages/balance-overview/balance-overview.component.ts [hunk 2]
```diff
@@ -1,8 +1,30 @@
...
+    
+  }
+
+  getFunnyImage(balance: number | undefined): string {
+    if (!balance) return './assets/images/3d-empty-box.png'; 
+    if (balance < 500) return './assets/images/3d-dog-from-behind.png'; 
+    if (balance < 1000) return './assets/images/3d-sitting-dog.png'; 
+    if (balance < 1500) return './assets/images/3d-dog-with-leash.png';
+    if (balance < 2000) return './assets/images/3d-dog-with-bag.png';
+    if (balance < 5000) return './assets/images/3d-man-playing-with-dog.png';
```
Comment (Macavity) [inline apps/client/src/app/pages/balance-overview/balance-overview.component.ts:27]: This block looks like it always wanted to be a `switch` block 😅
```suggestion
    switch(true){
        case balance < 500:
            return '';
        // ..
    }
```

File: apps/client/src/app/services/user.service.ts [hunk 3]
```diff
@@ -1,34 +1,81 @@
...
-    const user = this.users.find(user => user.name === username);
+    this.users$.subscribe(users => {
+      const user = users.find(user => user.name === username);
     this.currentUserSubject.next(user);
+    });
   }
 
   setUser(user: User) {
+    console.log("Storing selected user in local storage: ", JSON.stringify(user));
+    localStorage.setItem('selectedUser', JSON.stringify(user));
```
Comment (Macavity) [inline apps/client/src/app/services/user.service.ts:50]: [later] We should add a service to centralize local storage handling 💡

File: apps/client/src/app/services/user.service.ts [hunk 4]
```diff
@@ -1,34 +1,81 @@
...
+    console.log("Setting user:", user);
     this.currentUserSubject.next(user);
   }
+
+  getAvatarForUser(user: User | null): string {
```
Comment (Macavity) [inline apps/client/src/app/services/user.service.ts:55]: This is strange, we have `user.avatarUrl` for this.

**Review by Macavity — APPROVED** [review]
Some ideas, but in general looks great! 👍

File: apps/client/src/app/app.module.ts (hunk 1 above)
Comment (LisaPMunich) [reply apps/client/src/app/app.module.ts:34]: Angular does not support accessing environment variables from .env or .env.local files directly like Node.js does. I could handle it in the environment.ts and environment.prod.ts, but then it is not in sync with the backend. I could also write a script to sync the values. Is that really necessary to have those two in sync?

File: apps/client/src/app/services/user.service.ts (hunk 3 above)
Comment (LisaPMunich) [reply apps/client/src/app/services/user.service.ts:50]: Maybe. Here I did add it to the user.service on purpose, because it concerns the user.

Comment (Macavity) [reply apps/client/src/app/services/user.service.ts:50]: Let's create a ticket and solve this at a later point.

File: apps/client/src/app/app.module.ts (hunk 1 above)
Comment (Macavity) [reply apps/client/src/app/app.module.ts:34]: Doesn't need to, `process.env.APP_LOCALE` will be inserted by webpack (node.js) during compilation time. At least that's how it is usually working 🤔

Comment (Macavity) [reply apps/client/src/app/app.module.ts:34]: But we can do that later :)

**Pull request merged**